docs/data_access.rst

```diff
@@ -2,62 +2,124 @@
 Data Access
 =======================================================================================
 
-We publish the PUDL pipeline outputs in several ways to serve
-different users and use cases. We're always trying to increase the accessibility of the
-PUDL data, so if you have a suggestion, please `open a GitHub issue
+We publish the PUDL pipeline outputs in several ways to serve different users and use
+cases. We're always trying to increase the accessibility of the PUDL data, so if you
+have a suggestion, please `open a GitHub issue
 <https://github.com/catalyst-cooperative/pudl/issues>`__. If you have a question, you
-can `create a GitHub discussion <https://github.com/orgs/catalyst-cooperative/discussions/new?category=help-me>`__.
+can `create a GitHub discussion
+<https://github.com/orgs/catalyst-cooperative/discussions/new?category=help-me>`__.
 
-PUDL's primary data output is the ``pudl.sqlite`` database. All the tables are also
-distributed as individual `Apache Parquet <https://parquet.apache.org/docs/>`__ files
-which are more space efficient, have richer
-data types and are better suited for distributed and large-scale data analysis.
-We recommend working with tables with the ``out_`` prefix, as these tables contain
-the most complete and easiest to work with data. For more information about the
-different types of tables, read through :ref:`PUDL's naming conventions <asset-naming>`.
-
-.. _access-modes:
+We recommend working with tables with the ``out_`` prefix, as these tables contain the
+most complete and easiest to work with data. For more information about the different
+types of tables, read through :ref:`PUDL's naming conventions <asset-naming>`.
 
 ---------------------------------------------------------------------------------------
-How Should You Access PUDL Data?
+Quick Reference
 ---------------------------------------------------------------------------------------
 
-We provide six primary ways of interacting with PUDL data. Here's how to find out
-which one is right for you and your use case.
-
 .. list-table::
    :widths: auto
    :header-rows: 1
 
-   * - Access Method
-     - Types of User
+   * - Platform
+     - Format
+     - Version
```

Comment:
Since there was a fair bit of overlap between the different access methods we were describing before, I re-organized this documentation around data platform, format, and version, and added these columns to the reference table. I think it provides a clearer and more complete picture of what the options are, but doesn't also increase the number of options we're showing people.